tools/mo/openvino/tools/mo/graph/graph.py

```diff
@@ -988,6 +988,21 @@ def pseudo_topological_sort(self, reverse: bool = False):
         else:
             return list(reversed(order))
 
+    def pseudo_topological_sort_with_start_node(self, start_node: Node, reverse: bool = False):
+        nodes_without_inputs = [start_node.soft_get('name')]
+        order = list()
+        visited = set()
+        for node_name in nodes_without_inputs:
```

Comment:
why do we need `for` if it seems only one iteration always? Please consider simplify this function.

Reply:
You are right. Done.